src/NServiceBus.TransportTests/When_failure_happens_after_isolated_send.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.TransportTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using Transport;
+
+    public class When_failure_happens_after_isolated_send : NServiceBusTransportTest
+    {
+        //[TestCase(TransportTransactionMode.None)] - not relevant
```

Comment:
I think we can tests for None and Receive only as well since they should behave like this "by definition"?